src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar__block calendar__block--1 calendar__block--1--sun"></div>
```

Comment:
```suggestion
      <div class="calendar__block"></div>
```
remove modificators for block

use `:nth-child()` instead of `--1`

also, modificator like `--sun` should be added to `calendar`, not to the `calendar__block` element

the same with `calendar__block--last` - redundant:
![image](https://github.com/mate-academy/layout_calendar/assets/86923089/afeef962-0653-4990-921f-14df482d0f57)

use CSS combinators and `:nth-child()` to solve it